optuna/multi_objective/study.py

```diff
@@ -118,6 +118,45 @@ def load_study(
 ) -> "multi_objective.study.MultiObjectiveStudy":
     """Load the existing :class:`MultiObjectiveStudy` that has the specified name.
 
+    Example:
+
+        .. testsetup::
+
+                import os
+
+                if os.path.exists("example.db"):
+                    raise RuntimeError("'example.db' already exists. Please remove it.")
+
+        .. testcode::
+
+            import optuna
+
+            def objective(trial):
+                # Binh and Korn function.
+                x = trial.suggest_float("x", 0, 5)
+                y = trial.suggest_float("y", 0, 3)
+
+                v0 = (4 * x) ** 2 + (4 * y) ** 2
+                v1 = (x - 5) ** 2 + (y - 5) ** 2
+                return v0, v1
+
+            study = optuna.multi_objective.create_study(
+                directions=["minimize", "minimize"],
+                study_name="my_study",
+                storage="sqlite:///example.db"
+                )
```

Comment:
What do you think about applying `black` in this code? Since existing testcodes don't necessarily comply to `black`, you don't have to but just a suggestion.

Reply:
Thanks as always!
For simplicity and visibility,  testcodes that comply to `black` seems better.